ui/lib/apps/OptimizerTrace/index.meta.ts

```diff
@@ -0,0 +1,8 @@
+import translations from './translations'
+
+export default {
+  id: 'optimizer_trace',
+  routerPrefix: '/optimizer_trace',
+  translations,
+  reactRoot: () => import(/* webpackChunkName: "app_optimizer_trace" */ '.'),
```

Comment:
We don't need the `webpackChunkName` comment now.

Reply:
Gotcha!